### ISSUE TITLE
flock unavailable while opening fragment on 1.2

### ISSUE THREAD

**Issue by jaffee** [pilosa/pilosa#1810]
Similar to #1669 but probably not the same root cause since it's happening on 1.2.

While importing:
2019/01/02 15:57:44 pilosa.go:195: starting field import for test_bsi_field: importing values to nodes: doing import: Server error 500 Internal Server Error body:’importing existence columns: creating view: opening fragment: opening storage: flock: resource temporarily unavailable

Disabling the existence field seems to stop the problem.

This is on a 5 node cluster running imports from the pdk.

**Comment by jaffee** [comment]
I just got this with existence disabled:

```2019/01/14 15:46:54 starting field import for duration_minutes: importing columns to nodes: doing import: Server error 500 Internal Server Error body:'importing existence columns: creating fragment: opening fragment: opening storage: flock: resource temporarily unavailable
'
2019/01/14 15:46:54 starting field import for drop_grid_id: importing columns to nodes: doing import: Server error 500 Internal Server Error body:'importing existence columns: creating fragment: opening fragment: opening storage: flock: resource temporarily unavailable
'
2019/01/14 15:46:54 starting field import for pickup_elevation: importing columns to nodes: doing import: Server error 500 Internal Server Error body:'importing existence columns: creating fragment: opening fragment: opening storage: flock: resource temporarily unavailable```

**Comment by travisturner** [comment]
@jaffee that seems to imply that the default index options are being used at some point (by default `trackExistence: true`). Are you able to determine at which point the `_exists` directory is being created?

I just tried a basic import and wasn't able to reproduce that behavior. Can you describe the steps that happen prior to this error?

**Comment by jaffee** [comment]
OK, so it seems like go-pilosa 1.1 had a bug that didn't respect passing the option to explicitly disable existence tracking. Updating the PDK go-pilosa dependency (to `00165520cfce`) fixed this issue. 

Probably what we're back to now is that this issue only occurs when existence is enabled

**Comment by jaffee** [comment]
believe this was fixed by #1863
